README.md

```diff
@@ -122,11 +123,14 @@ Suricata is a free intrusion detection/prevention system and network security mo
 
 - [SELKS](https://github.com/StamusNetworks/SELKS) - A Suricata based IDS/IPS/NSM distro.
 - [Amsterdam](https://github.com/StamusNetworks/Amsterdam) - Docker based Suricata, Elasticsearch, Logstash, Kibana, Scirius aka SELKS.
+- [pfSense](https://www.pfsense.org/) - Is a free network firewall distribution, based on the FreeBSD operating system with a custom kernel and including third party free software packages for additional functionality.
```

Comment:
Could you remove the trailing slash here and replace the "Is a" in the description with "A", so it becomes "A free network firewall distribution, ..."?

Please do the same for OPNsense -- thanks!

Reply:
Done

Reply:
Thanks! Merging...